mani_skill/utils/building/actors.py

```diff
@@ -344,3 +344,29 @@ def build_actor_ai2(
     if set_object_on_ground:
         actor.set_pose(sapien.Pose(p=[0, 0, 0]))
     return actor
+
+def build_actor_ai2_helper(
```

Comment:
i think we will discuss how to unify the building API in the future so we might change this function